src/App.jsx

```diff
@@ -1,29 +1,39 @@
-import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import './App.css';
-import Header from './Header';
-import Questions from './Questions';
-import Answer from './Answer';
-import QuestionData from './Questions.json';
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <div className="Content">
-          <Header />
-          <h2>Manager is ready to help you!</h2>
-          <Route exact path="/" component={Questions} />
-          <Route
-            path="/question/:questionId"
-            render={({ match }) => (<Answer data={QuestionData.find(item =>
-            item.id.toString() === match.params.questionId)}
-            />
-       )} />
-        </div>
-      </Router>
-    );
-  }
-}
-
-export default App;
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import './App.css';
+import Header from './Header';
+import Questions from './Questions';
+import Answer from './Answer';
+import QuestionData from './Questions.json';
+
+class App extends Component {
```

Comment:
I mean that when you see name like App and you go inside the component you don't expect to see Router and routing logic. In my opinion, it's semantically incorrect.